docs/disaster-recovery/modules/cosmosdb.md

```diff
@@ -0,0 +1,286 @@
+# [Azure CosmosDB](https://azure.microsoft.com/en-us/services/cosmos-db/)
```

Comment:
The name "Cosmos DB" has a blank space. It is "DocumentDB" and "MongoDB" who doesn't have a blank. Let's make it more accurate.

Reply:
Will change it. Btw, in OBSA [doc](https://github.com/Azure/open-service-broker-azure/blob/master/docs/modules/cosmosdb.md), it uses `CosmosDB` without space, should we open an issue reporting it?

Reply:
Yes. Better it is also fixed in OSBA side. To be honest, I think maybe fixing it in titles is enough. In text, CosmosDB looks good.